src/modules/common/components/ScrollingSearchBox.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import style from './ScrollingSearchBox.css'
+import { animateScroll } from 'react-scroll'
+import SearchInput from './SearchInput'
+
+export class ScrollingSearchBox extends React.Component {
+  static propTypes = {
+    filterText: PropTypes.string.isRequired,
+    onFilterTextChange: PropTypes.func.isRequired,
+    stickyTop: PropTypes.number.isRequired,
+    spaceSearch: PropTypes.bool,
+    children: PropTypes.element.isRequired
+  }
+
+  static defaultProps = {
+    stickyTop: 0
```

Comment:
stickyProps is always default, why do we need it? looks more like a constant.

Reply:
StickyTop is the top position of the DOM-element where the element should be fixed when scrolling below. If you have multiple sticky elements (in this case SearchInput and the selector group initials), we need to tell our child components (which could also have a sticky element) what the most top position should be. (If you always set stickyTop: 0, then the groups would be hidden by the search input)